lib/checking_account.rb

```diff
@@ -0,0 +1,55 @@
+require_relative 'account'
+
+module Bank
+  class CheckingAccount < Account
+
+    def initialize(id, balance)
+      super(id, balance)
+      @number_of_checks = 0
+    end
+
+    def withdraw(amount) #I'm doing something with super. So just copied the whole thing.
```

Comment:
As you indicated in your PR, it's definitely possible to DRY this up using `super`. In this case `Account#withdraw` would probably require more parameters, like a fee and a minimum balance.